ceph.spec.in

```diff
@@ -468,6 +469,9 @@ BuildRequires:  libnuma-devel
 %if 0%{?rhel} >= 8
 BuildRequires:  /usr/bin/pathfix.py
 %endif
+%if 0%{with mgr_dashboard_frontend_support_multi_language}
+BuildRequires:  npm
```

Comment:
Maybe that works as it is. If not, as we're now in Angular 15, [that might require specific versions](https://angular.io/guide/update-to-version-15):
```suggestion
BuildRequires:  node >= 14.20, node>=16.13, node >= 18.10, node<20.0
BuildRequires:  npm
```

Reply:
When I saw the following screenshot message, I found that we must limit the version of npm.
![image](https://github.com/ceph/ceph/assets/50572318/7f4e9aaa-5dbd-41b2-b895-090242040761)